src/app/staticanalysis/modules/functionCallGraph.js

```diff
@@ -16,6 +16,30 @@ function buildLocalFuncCallGraphInternal (functions, nodeFilter, extractNodeIden
   return callGraph
 }
 
+/**
+ * Builds a function call graph for the current contracts.
+ * Example Contract call graph:
+ *
+ * {
+ *  "KingOfTheEtherThrone": {
+ *    "contracts": {...},                                        // Contract node as defined in abstractAstView.js
+ *    "functions": {
+ *      "KingOfTheEtherThrone.claimThrone(string memory)": {    // function in KingOfEtherThrone
```

Comment:
Are you sure about that? I just tried to define two contracts with the same name... got an Error that the identifier is already declared. Can you give me an example of where this would work\make sense?